google/earthenterprise
# Fixes #200 (using symbolic/soft links instead of hard links)

Fixes:
- #200 

To verify:
- Move `/tmp` to different partition (`sudo systemctl enable tmp.mount` and reboot).
- Build and `stage_install` using scons.
- `ls -la /tmp/fusion_os_install/tutorial/opt/google/share/tutorials/fusion/*` should now show properly connected symbolic links.
